openapi-v2.yaml

```diff
@@ -764,6 +764,14 @@ paths:
         in: query
         description: "Show only the credentials from this provider. This should be one of the CloudProvider enum values."
         type: string
+      - name: page
+        in: query
+        description: pagination offset
```

Comment:
Is the unit for this "pages", as in when querying we should skip `page * per_page` units before starting, or is it just the number of units?

It would probably be good to add this detail to the `PaginationMetadata` object.

Reply:
`page` is the offset and `per_page` is the limit. So we should include something like this in the query: `query.Offset((page - 1) * per_page).Limit(per_page)`
Both properties are described in the [PaginationMetadata definition](https://github.com/TileDB-Inc/TileDB-Cloud-API-Spec/blob/master/openapi-v2.yaml#L509)